src/Cli/InstallPluginCommand.php

```diff
@@ -53,6 +91,23 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $outputStyle->writeln('<info>Installing RBAC plugin...</info>');
 
         foreach ($this->commands as $step => $command) {
+            if ($this->isGrantAccessToGivenAdministratorCurrentCommand($command['command'])) {
+                if (!$this->isAdministratorEmailProvided()) {
+                    continue;
+                }
+
+                $command['parameters']['sections'] = $this->syliusSectionsProvider->__invoke();
+                $command['parameters']['administratorEmail'] = $this->administratorEmail;
```

Comment:
I would extract this and next `if` statement body to a private function, for better readability